conftest.py

```diff
@@ -2,13 +2,22 @@
 
 import pytest
 from django.conf import settings
+from django.core.files.base import ContentFile
 from django.test import Client
+from django.test.utils import override_settings
 from wagtail.models import Page
 
+from cms.documents.models import CustomDocument
 from cms.home.models import HomePage
 from cms.release_calendar.models import ReleaseCalendarIndex, ReleaseCalendarPage
 
 
+@pytest.fixture(scope="session", autouse=True)
+def _custom_settings(tmpdir_factory):
+    overrides = override_settings(MEDIA_ROOT=str(tmpdir_factory.mktemp("media")))
+    overrides.enable()
+
+
```

Comment:
Question: is it possible to do this as a context manager? So the settings are properly disabled

```suggestion
@pytest.fixture(scope="session", autouse=True)
def _custom_settings(tmpdir_factory):
    with override_settings(MEDIA_ROOT=str(tmpdir_factory.mktemp("media"))):
        yield


```

Reply:
pushed these to #16. thanks, this works